pandas/tests/extension/base/setitem.py

```diff
@@ -32,22 +34,25 @@ def test_setitem_sequence_mismatched_length_raises(self, data, as_array):
         xpr = 'cannot set using a {} indexer with a different length'
         with tm.assert_raises_regex(ValueError, xpr.format('list-like')):
             ser[[0, 1]] = value
+            assert ser._values[[0, 1]] == value
```

Comment:
@TomAugspurger how will this assert be reached if the previous line raises an error?

Reply:
Ah, sorry I missed that.